src/auto/injector.js

```diff
@@ -41,7 +41,8 @@
  * In the following example a new block of HTML containing a `ng-controller`
  * directive is added to the end of the document body by JQuery. We then compile and link
  * it into the current AngularJS scope.
- *
+ * Note: The below code works only when debug info is enabled. (see {@link api/ng/function/angular.injector debugInfoEnabled}).
```

Comment:
Could you please refactor this to `<div class="alert alert-info">**Note:** The below ...</div>`?